cli/src/semgrep/config_resolver.py

```diff
@@ -125,7 +125,8 @@ def __init__(
             self._supports_fallback_config = True
         elif is_policy_id(config_str):
             state.metrics.add_feature("config", "policy")
-            self._config_path = url_for_policy()
+            self._origin = ConfigType.SEMGREP_CLOUD_PLATFORM
+            self._config_path = config_str
             self._supports_fallback_config = True
```

Comment:
For the time being I think we have to set this to false since the url format / access pattern changes and isn't compatible with this hacky fallback logic I added a while back.
```suggestion
            self._supports_fallback_config = False
```

Reply:
In the [PR](https://github.com/semgrep/semgrep/pull/9205/files#), the one this PR depends on, I use the [old URL to construct the fallback URL](https://github.com/semgrep/semgrep/pull/9205/files#diff-160acc92ed28ed2db5b7e3b73325a96deb5eb4b6f1b346631026f9c75b3a86e3R371). Wouldn't this still work?